### ISSUE TITLE
Failed to fetch

### ISSUE THREAD

**Issue by LoffeKoffe** [citizenfx/screenshot-basic#14]
![image](https://user-images.githubusercontent.com/46104331/69179452-d982df80-0b0b-11ea-81a8-544bb500986f.png) 

I'v done some research about this error which make me beleve that "https://wew.wtf/upload.php" cant be reached. But i dont really know, I just know that some of my projects depend on this working:)

**Comment by ridikacool1** [comment]
Same problem here, updated cfx and artifact still the same.

**Comment by LoffeKoffe** [comment]
Yeah i did that too. I have tried a few possible fixes without any luck. Like I said its probably wew.wtf that isnt availible anymore sadly

**Comment by WalterWhite84** [comment]
look a post for help bro : https://forum.fivem.net/t/api-dev-resource-screenshot-basic-make-screenshots-of-clients-games/234165/96

**Comment by LoffeKoffe** [comment]
Thank you!

**Comment by majidmazaheri** [comment]
did you find the fix?
I have the same issue

**Comment by AgenChicken24** [comment]
Did anybody fixed this issue?
